pandas/tools/tile.py

```diff
@@ -81,6 +83,17 @@ def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
     array([1, 1, 1, 1, 1], dtype=int64)
     """
     # NOTE: this binning code is changed a bit from histogram for var(x) == 0
+    # for handling the cut for datetime and timedelta objects
+
```

Comment:
Thx for the change. I **think** ``_preprocess_for_cut`` should be called before ``_coerce_to_type`` to handle ``list`` properly?